### PR TITLE
added missing description

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9360

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

**Comment by MaratMussabekov** [comment]
Dear @e0i, for your information

Thank you

File: windows/client-management/mdm/vpnv2-csp.md [hunk 1]
```diff
@@ -866,6 +866,17 @@ Added in Windows 10, version 1607. Specifies the class-based default routes. Fo
 
 Value type is bool. Supported operations include Get, Add, Replace, and Delete.
 
+<a href="" id="vpnv2-profilename-nativeprofile-l2tppsk"></a>**VPNv2/**<em>ProfileName</em>**/NativeProfile/PlumbIKEv2TSAsRoutes**  
+Determines whether plumbing IPSec traffic selectors as routes onto VPN interface is enabled.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/vpnv2-csp.md:870]:
```suggestion
Determines whether plumbing IPSec traffic selectors as routes onto VPN interface are enabled.
```

File: windows/client-management/mdm/vpnv2-csp.md [hunk 2]
```diff
@@ -866,6 +866,17 @@ Added in Windows 10, version 1607. Specifies the class-based default routes. Fo
...
+
+If set to False, plumbing traffic selectors as routes is disabled
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/vpnv2-csp.md:872]:
```suggestion
If set to False, plumbing traffic selectors as routes are disabled.
```

File: windows/client-management/mdm/vpnv2-csp.md [hunk 3]
```diff
@@ -866,6 +866,17 @@ Added in Windows 10, version 1607. Specifies the class-based default routes. Fo
...
+
+If set to True, plumbing traffic selectors as routes is enabled
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/vpnv2-csp.md:874]:
```suggestion
If set to True, plumbing traffic selectors as routes are enabled.
```

File: windows/client-management/mdm/vpnv2-csp.md [hunk 4]
```diff
@@ -866,6 +866,17 @@ Added in Windows 10, version 1607. Specifies the class-based default routes. Fo
...
+
+By default, this value is false.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/vpnv2-csp.md:876]:
```suggestion
By default, this value is set to False.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by illfated** [comment]
@JohanFreelancer9 : I must admit that I find the suggestion to change from "plumbing traffic selectors as routes **is** enabled" to "plumbing traffic selectors as routes **are** enabled" more difficult to understand than most other uncertain grammatical scenarios. When I read those sentences, I read them as meaning that **plumbing** is enabled, not "traffic selectors" are enabled or "routes" are enabled.

Have I missed an important way of interpreting those sentences?

File: windows/client-management/mdm/vpnv2-csp.md (hunk 2 above)
Comment (JohanFreelancer9) [inline windows/client-management/mdm/vpnv2-csp.md:872]:
```suggestion
If set to False, plumbing traffic selectors as routes is disabled.
```

**Comment by JohanFreelancer9** [comment]
@illfated - I take your point and I have misinterpreted these sentences. Thanks for pointing it out. I have changed my suggestions accordingly.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**